test/test_helper.rb

```diff
@@ -100,3 +118,33 @@ def call(name, start, finish, message_id, values)
 
   ActiveSupport::Notifications.subscribe('sql.active_record', SQLCounter.new)
 end
+
+module Kernel
+  def quietly_with_deprecation_silenced(&block)
+    ActiveSupport::Deprecation.silence do
+      quietly_without_deprecation_silenced(&block)
+    end
+  end
+  alias_method_chain :quietly, :deprecation_silenced
+end
+
+# UPGRADE
+# Assert queries was removed as of rails 4.0.2
+class ActiveSupport::TestCase
```

Comment:
If possible and straightforward, I have a slight preference for preserving `ActiveRecord::TestCase` (by declaring it here if necessary) instead of polluting `ActiveSupport::TestCase`